kyo-combinators/shared/src/main/scala/kyo/internal/Zippable.scala

```diff
@@ -0,0 +1,23 @@
+package kyo.internal
+
+trait Zippable[-A, -B]:
```

Comment:
I think we can use abstract classes for all the traits here.
```suggestion
abstract class Zippable[-A, -B]:
```